src/interceptors/terminal/terminal-scripts.ts

```diff
@@ -124,8 +124,31 @@ export const getFishShellScript = (callbackUrl: string, env: { [name: string]: s
     echo 'HTTP Toolkit interception enabled'
 `;
 
-export const getPowerShellScript = (callbackUrl: string, env: { [name: string]: string }) => `${
-        _.map(env, (value, key) => `    $Env:${key} = "${value.replace(/"/g, '`"')}"`).join('\n')
+export const getPowerShellScript = (callbackUrl: string, env: { [name: string]: string }) => `
+    $HTTPTOOLKIT_envVars = Get-ChildItem Env:
+
+${
+    _.map(env, (value, key) => `    $Env:${key} = "${value.replace(/"/g, '`"')}"`).join('\n')
+}
+
+    function Stop-Toolkit {
```

Comment:
Can we rename this to `Stop-Intercepting`? Or I'm open to other ideas too (Detach/Disconnect could work, or maybe Stop-Intercept_ion_ or something) but regardless I think 'Toolkit' isn't quite right (it's a bit too generic a term by itself, it's not actually stopping HTTP Toolkit - it's just disconnecting this terminal - and I think the command should really describe what the terminal does rather than the tool you're using elsewhere).

Reply:
Yeah sure Stop-Intercepting sounds great